src/js/login/containers/Main.jsx

```diff
@@ -75,6 +80,7 @@ class Main extends React.Component {
       });
       const receiver = window.open(`${myLoginUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
       receiver.focus();
+      this.getLoginUrl();
```

Comment:
@cvalarida I also noticed the refresh happening in `componentDidMount()`, which means that the [AuthN URL Re-Use](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1345) must be occurring by way of someone clicking "Sign In" repeatedly w/o refreshing the page - this PR addresses that use-case.

I have a feeling that we may not have heard the last of this and that the URLs are being re-used in some other unknown way, especially when you consider that one URL was used over the course of 30+ days!  I will followup with ID.me.